src/stories/components/Bar/DodgedBar.story.tsx

```diff
@@ -86,6 +86,7 @@ const Color = bindWithProps(DodgedBarStory);
 Color.args = {
 	type: 'dodged',
 	dimension: 'browser',
+	onClick: undefined,
 	order: 'order',
 	color: 'operatingSystem',
 };
```

Comment:
Can you do something like we did in `Bar.test` where you put all the props that are shared across all stories and put them in const const called `defaultProps` and then spread them across each story. Just cleans things up a bit.